main.js

```diff
@@ -0,0 +1,11 @@
+const question = prompt( '日本の首都は?' );
+
+while ( true ) {
+  if (question === '東京') {
+    alert ( '正解です!' );
+    break;
+  } else {
+    alert ( '不正解です!' );
+  }
+}
```

Comment:
すみません、ちょっと私の伝え方が悪かったです。
これだと、
- 正解した時にループが終わる
- 不正解だと、ずっと「不正解です」のループになるかと思います。

なので、promptの定義からループさせるようにすると、質問の正誤に関わらず質問からループさせることが出来ます。

コードも載せておきますので、参考になさってください。
```suggestion
while ( true ) {
  const question = prompt( '日本の首都は?' );
  if (question === '東京') {
    alert ( '正解です!' );
  } else {
    alert ( '不正解です!' );
  }
}
```